src/azure-cli/azure/cli/command_modules/keyvault/vendored_sdks/azure_keyvault_securitydomain/_internal/async_challenge_auth_policy.py

```diff
@@ -32,11 +33,6 @@
 from . import http_challenge_cache as ChallengeCache
 from .challenge_auth_policy import _enforce_tls, _has_claims, _update_challenge
 
-if sys.version_info < (3, 9):
-    from typing import Awaitable
-else:
-    from collections.abc import Awaitable
```

Comment:
@evelyn-ys, let's make sure the new SDK no longer contains such code.

Reply:
cc @mccoyp

Reply:
The Python SDK supports Python versions until 6 months after official end of support from PSF ([version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)). Since 3.8 went EOL this October, we'll drop support in April 2025. At that point we can remove conditional imports like this one in the SDK.